package/MDAnalysis/coordinates/chain.py

```diff
@@ -47,6 +47,7 @@
 import os.path
 import itertools
 import bisect
+import copy
```

Comment:
You don't need to import `copy` here, we're not using it.  `copy` would be needed if you called `copy.something()` not for calling `thing.copy()`